### PR TITLE
[RFC] README.md: Link to neovim.org/Community

### PR DESCRIPTION
**Pull request by ghost**
This brings it more in line with the mentioned wiki page. The linked wiki page is tiny, so it might be better to just move it into the README.

rendered: https://github.com/Pyrohh/neovim/blob/readme-patch/README.md

### CODE REVIEW

**Comment by fwalch** [comment]
Also see neovim/neovim.github.io#91.

**Comment by Pyrohh** [comment]
@fwalch now that your PR is merged, what do you think about just linking to `http://neovim.org/community/`? I could also update the wiki's `Communicating.md` to reference it.

**Comment by fwalch** [comment]
> @fwalch now that your PR is merged, what do you think about just linking to http://neovim.org/community/? I could also update the wiki's Communicating.md to reference it.

:+1:

**Comment by Pyrohh** [comment]
repushed

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ more. See [the wiki](https://github.com/neovim/neovim/wiki/Installing-Neovim)!
 
 ### Community
```
Comment (justinmk) [inline README.md]: Let's remove this section entirely and put the Community link immediately right of the "Website" link at the top.

Comment (justinmk) [inline README.md]: And remove the Reddit link, since it's on the Community page and is not important enough to be at the top of the readme.

Comment (Pyrohh) [inline README.md]: Twitter and the mailing list are already mentioned at the Community page, so maybe those should be removed too?

Comment (justinmk) [inline README.md]: They're more important, they can stay until we need the room for something else.

Comment (Pyrohh) [inline README.md]: Also those links up top might be easy to miss, so perhaps they should be made bold, or have their size increased a little. Really anything to make them stand out.

**Comment by Pyrohh** [comment]
repushed, if this looks good to you @justinmk I'll merge it.

**Comment by justinmk** [comment]
:+1:

**Comment by Pyrohh** [comment]
merged, thanks

**Pull request closed without merging**